.github/workflows/jest-coverage-changes.yml

```diff
@@ -0,0 +1,26 @@
+name: Jest Coverage - changed files
+
+on:
+  pull_request:
+    branches:
+      - develop
+      - main
```

Comment:
do we need coverage check on main and release PRs ?

Reply:
removed and restricted to `develop`